docs/source/tutorials/datalab/datalab_quickstart.ipynb

```diff
@@ -746,6 +774,60 @@
     "\n",
     "To learn more, check out this [example notebook](https://github.com/cleanlab/examples/blob/master/datalab_image_classification/datalab.ipynb) (demonstrates Datalab applied to a real dataset) and the [advanced Datalab tutorial](datalab_advanced.html) (demonstrates configuration and customization options to exert greater control)."
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "metadata": {}
+   },
+   "outputs": [],
+   "source": [
+    "issue_results = lab.get_issues(\"label\")\n",
+    "outlier_results = lab.get_issues(\"outlier\")\n",
+    "duplicate_results = lab.get_issues(\"near_duplicate\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "metadata": {}
+   },
+   "outputs": [],
+   "source": [
+    "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "from sklearn.metrics import roc_auc_score\n",
+    "\n",
+    "def jaccard_similarity(l1, l2):\n",
+    "    s1 = set(l1)\n",
+    "    s2 = set(l2)\n",
+    "    intersect_set = s1.intersection(s2)\n",
+    "    union_set = s1.union(s2)\n",
+    "    if len(intersect_set) == 0:\n",
+    "        return 0\n",
+    "    return len(intersect_set) / len(union_set)\n",
+    "\n",
+    "identified_label_issues_indices = issue_results[issue_results[\"is_label_issue\"] == True].index.tolist()\n",
+    "label_issue_indices = issue_results[issue_results[\"given_label\"] != issue_results[\"predicted_label\"]].index.tolist()\n",
```

Comment:
The `given_labels` are noisy, so testing the Jaccard similarity doesn't quite make sense here.

The `create_data()` function defined in the notebook should have "ground truth" labels that you can compare against.

Reply:
I am now using y_train_idx as ground labels and noisy_labels_idx as predictions from create_data() function for comparision.